server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -184,9 +184,8 @@ public static String switchRootTableVolume(String location) throws IOException {
    * for use it chooses a new tablet directory.
    */
   public static TabletFiles updateTabletVolumes(AccumuloServerContext context, ZooLock zooLock,
-      VolumeManager vm, KeyExtent extent, TabletFiles tabletFiles, boolean replicate)
-      throws IOException {
-    List<Pair<Path,Path>> replacements = ServerConstants.getVolumeReplacements();
```

Comment:
Would it make more sense to keep the replacements assignment in this method, but short-circuit the method with a `if (replacements.isEmpty()) { return; }` line?

It seems like this is a smaller change and would keep the logic all together that handles the updates, rather than place the special case of no replacements outside the method.

Reply:
Well we wouldn't be able to just return because the return value would wipe out the files and logs kept in tabletPaths.  But since that object gets passed I think we could just return it.

Reply:
+1

Reply:
Oh, right. I didn't see there was a return value.